<launch>
  <node pkg="my_image_transport" name="my_image_transport_node" type="my_image_transport_node" output="screen">
  <param name="advertise_topic_name" type="string" value="/fisheye/image"/>
  <param name="video_location" type="string" value="/home/kevin/workspace/IARC/video/bottom_view_0826.avi"/>
  <param name="show_video" type="bool" value="true"/>
  <param name="loop_rate_num" type="int" value="35"/>
  </node>
</launch>
